### ISSUE TITLE
dynamic sizing

### ISSUE THREAD

**Issue by frigon** [Telerik-Verified-Plugins/Mapbox#1]
Eddy -firstly I love this plugin.

This is less of an issue than a question:
Are there any examples of using this in conjunction with kendo mobile... i'm trying to get the map to appear as though its inline with the kendo header and footer... e.g. not a popup. Since the header/footer size in kendo would adjust based on platform and device I can't hardcode a value into the size of the mapbox in javascript. 

In short I need to dynamically size the map to make it appear inline. 

Any recommendations?

**Comment by frigon** [comment]
FYI - I resorted to using some trickery on $("header").height() to get this to work

**Comment by EddyVerbruggen** [comment]
Can you perhaps share it for others (until perhaps we add something native and xplatform)?

**Comment by frigon** [comment]
i'm using kendo so on the views show event I then open the map and grab the header and footer size via:

Mapbox.show({
    ...
    ...
    margins: {
         left: 0,
         right: 0,
         top: $("#my-view header").height(),
         bottom: $("#my-view footer").height()
    },
    ...
});

Could be done without kendos view show event with just jquery though.

**Comment by rynangeles** [comment]
what if my options are in a sidebar currently using it with ionic how can i show sidebar on top of the map?

**Comment by josetaira** [comment]
Using ionic sidemenu as well. Currently, opening the sidemenu shows that sidemenu below the map.

**Comment by EddyVerbruggen** [comment]
Regarding the side menu: what if I'd add an option to move the map below the webview (and making the webview transparent so you can see the map)? When you open the drawer you move the map to the back, and when the drawer has been closed you can call a second function to make the map overlay the webview again (because if you'd leave it below the webview you can't interact with it (touchevent-wise)).

**Comment by josetaira** [comment]
@EddyVerbruggen, alternatively, the map could move along where the map container is. For the case of ionic sidemenu, when you open the sidemenu, the main content slides to the right to accommodate the sidemenu (it doesn't overlay the map). When this happens though, the map doesn't follow. Your suggestion would also work in the general case, although this one also has it's uses. For your consideration.

**Comment by josetaira** [comment]
@EddyVerbruggen any updates on this?

**Comment by diego086** [comment]
Hey Eddy!! 
First of all great plugin!!

Im having the same problem with the ionic sidemenu. Let me know if you implement a solution about this.

Thanks
